src/components/user-profile.html

```diff
@@ -57,34 +50,72 @@
         margin-top: 5px;
         padding-top: 10px;
       }
-
+      jha-icon-delete, jha-icon-edit, jha-icon-save, jha-icon-visibility {
+        fill:#fcfcff;
+        height: 16px; 
+        width: 16px; 
+        vertical-align: text-bottom; 
+        padding-left: 5px; 
+      }
+      jha-avatar {
+        padding-bottom: 10px; 
+        height: 48px; 
+        width: 48px;
+      }
       .delete {
         background-color: #d1250ee3;
       }
+      @media (max-width: 600px) {
+        table {
+          width: 200px;
+          padding-left: 5px;
+          display: contents;
+        }
+        td {
+          padding-left: 10px;
+          font-size: 11px;
+          vertical-align: bottom;
+        }
+        button {
+          background-color: indigo;
+          color: #fcfcff;
+          border-radius: 3px;
+          text-align: center;
+          padding: 8px;
+          margin-top: 10px;
+          font-size: 9px;
+          margin-left: 10px;
+          margin-right: 10px;
+          width: 110px;
+        }
+        .save {
+          display: none;
+        }
+      }
     </style>
     <template is="dom-if" if="[[addUserSelected]]">
       <table>
         <tr>
-          <td><label for="firstName">First name </label></td>
-          <td><input type="text" name="firstName" id="firstName" value="{{user.firstName::input}}"></td>
+          <td><label for="firstName">First name</label></td>
+          <td><input type="text" id="firstName" name="firstName" placeholder="First Name" value="{{user.firstName::input}}"></td>
         </tr>
         <tr>
-          <td><label for="lastName">Last name </label></td>
-          <td><input type="text" name="lastName" id="lastName" value="{{user.lastName::input}}"></td>
+          <td><label for="lastName">Last name</label></td>
+          <td><input aria-label="input" type="text" id="lastName" name="lastName" placeholder="Last Name" value="{{user.lastName::input}}"></td>
```

Comment:
the aria-label is what you want the screenreader to announce when someone focuses this input.. so in this case if you're going to add a label it should say "Last name"

Really though, you don't need to add the aria-label because using the label tag on line 103 is sufficient.  I would remove the aria-label here and on the rest of these inputs